specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Jobs.json

```diff
@@ -1075,12 +1075,69 @@
             "template": {
               "description": "Job's execution container.",
               "$ref": "#/definitions/JobExecutionTemplate"
+            },
+            "detailedStatus": {
+              "description": "Detailed status of the job execution.",
+              "$ref": "#/definitions/ExecutionJobStatus"
             }
           },
           "x-ms-client-flatten": true
         }
       }
     },
+    "ExecutionJobStatus": {
+      "description": "Container Apps Job execution.",
+      "type": "object",
+      "properties": {
+        "pods": {
+          "description": "Pods in the execution.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExecutionPodStatus"
+          }
+        }
+      }
+    },
+    "ExecutionPodStatus": {
+      "description": "Container Apps Job execution.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Pod Name.",
+          "type": "string"
+        },
+        "containers": {
+          "description": "Containers in the execution",
+          "type": "array",
```

Comment:
please add `x-ms-identifiers` for the array

Reply:
added